test/kokoro/xds.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -exu -o pipefail
+if [[ -f /VERSION ]]; then
+  cat /VERSION
+fi
+
+cd github
+
+export GOPATH=${HOME}/gopath
```

Comment:
`export GOPATH="${HOME}/gopath"`

Just in case `HOME` contains a space character.

Reply:
Done